src/pages/Waiting/index.js

```diff
@@ -1,77 +1,86 @@
-import React from "react";
+import React, { useState } from "react";
 import styled from "styled-components";
 
 export default function WaitingRoom() {
+  const [isSelected, setIsSelected] = useState(false);
+  const [isReady, setIsReady] = useState(false);
+  const toggleIsSelected = () => setIsSelected(!isSelected);
+  const toggleIsReady = () => setIsReady(!isReady);
+
+  const handleReady = (e) => {
```

Comment:
인자를 e로 하는 것 보다는 event 혹은 흔히 많이 쓰이는 ev로 해주는 것은 어떨까합니다. 
```suggestion
  const handleReady = (e) => {
```